public/homepage/scripts/main.js

```diff
@@ -1,4 +1,4 @@
-/* global requirejs */
+// global requirejs 
```

Comment:
I tried submitting the code with  /* global requirejs */ but Travis fails the push and stays it is an error since requirejs is declared and not used.